### ISSUE TITLE
Missing compression CV with "-z" and MSGF+

### ISSUE THREAD

**Issue by jpfeuffer** [compomics/ThermoRawFileParser#107]
Hi!

I understand that "-z" deactivates zlib compression (which I needed for XTandem).
However, then MSGF+ complains with:
```
Standard error: java.lang.IllegalStateException: Required compression CV parameter not found in BinaryDataArray!
	at uk.ac.ebi.jmzml.model.mzml.BinaryDataArray.needsUncompressing(BinaryDataArray.java:721)
	at uk.ac.ebi.jmzml.model.mzml.BinaryDataArray.getBinaryDataAsNumberArray(BinaryDataArray.java:314)
	at edu.ucsd.msjava.mzml.SpectrumConverter.getSpectrumFromJMzMLSpec(SpectrumConverter.java:168)
	at edu.ucsd.msjava.mzml.MzMLSpectraIterator.parseNextSpectrum(MzMLSpectraIterator.java:53)
	at edu.ucsd.msjava.mzml.MzMLSpectraIterator.<init>(MzMLSpectraIterator.java:26)
	at edu.ucsd.msjava.msutil.SpectraAccessor.getSpecItr(SpectraAccessor.java:89)
	at edu.ucsd.msjava.ui.MSGFPlus.runMSGFPlus(MSGFPlus.java:263)
	at edu.ucsd.msjava.ui.MSGFPlus.runMSGFPlus(MSGFPlus.java:113)
	at edu.ucsd.msjava.ui.MSGFPlus.main(MSGFPlus.java:61)
```

Maybe there needs to be a "compression = none" parameter if there is such a thing? Or there is a remaining attribute (despite "-z") somewhere that triggers "needsUncompressing" while it shouldn't?

I'll have a closer look at the mzML but maybe you know something right away.

Thanks!

**Comment by caetera** [comment]
Hi Julianus,

yes, you are correct. I checked once again the mzML specification, the compression definition is required, i.e. there should be MS:1000576 (no compression) when `-z` is used. It will be fixed. Thank you for reporting.

**Comment by nielshulstaert** [comment]
This is fixed in release 1.3.3
